chingu-coders/Voyage2-Turtles-02
# newsfeed settings options

## Purpose
* Adds advanced options for newsfeed

## Approach/Fixes
* Complies with advanced ffeatures mentioned in the newsfeed issue
* I also fixed a bug in the clock settings

## Testing
* Checkout this pull request.
* open the settings and check for three dropdown menus for selecting newssources
* select different source options, and check to make sure that the selected source is disabled in the other two dropdown menus (so that the user cant select the same source from all three dropdowns)
* check the newsfeed to see that sources have updated
